### DISCUSSION TITLE
Link Icon

### DISCUSSION THREAD

**Discussion by j-steinbach** [srid/emanote#135] [Ideas]
This might be a customizable HTML/CSS thing, but Emanote should add icons to local and external links. 

Benefits: 
- I see exactly which link leads to an internal note/node that holds a meaningful conversation and which I can edit and have control of.
- I see which link leads me to an external page which moves me out of my ZK.

See [this](https://note.ema.srid.ca/resources/vscode) emanote page for example. It should be signaled that the first link (Visual Studio Code) is external, whereas the second link (emanote-template) is internal.

This is also how it is done on Wikipedia (just check the sources footer), with an icon that is similar to [this](https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=external).

**Comment by srid** [comment]
Worth doing, https://github.com/srid/emanote/issues/136
